scripts/entity/tactical/enemies/rf_bandit_marauder.nut

```diff
@@ -0,0 +1,145 @@
+		this.getSprite("shield_icon").setBrightness(0.85);
+
+		this.m.Skills.add(this.new("scripts/skills/perks/perk_brawny"));
+		this.m.Skills.add(this.new("scripts/skills/perks/perk_rf_bully"));
+		this.m.Skills.add(this.new("scripts/skills/perks/perk_recover"));
+		this.m.Skills.add(this.new("scripts/skills/perks/perk_hold_out"));
+		this.m.Skills.add(this.new("scripts/skills/perks/perk_rf_survival_instinct"));
+	}
+
+	function onAppearanceChanged( _appearance, _setDirty = true )
+	{
+		this.actor.onAppearanceChanged(_appearance, false);
+		this.setDirty(true);
+	}
+
+	function assignRandomEquipment()
+	{
+		if (this.m.Items.hasEmptySlot(this.Const.ItemSlot.Mainhand))
+		{
+			local weapon = ::MSU.Class.WeightedContainer([
+	    		[1, "scripts/items/weapons/rf_battle_axe"],
+				[1, "scripts/items/weapons/rf_greatsword"],
+				[1, "scripts/items/weapons/two_handed_mace"],
+				[1, "scripts/items/weapons/two_handed_wooden_hammer"],
+				[1, "scripts/items/weapons/two_handed_wooden_flail"],
+				[1, "scripts/items/weapons/woodcutters_axe"]
+	    	]).roll();
+
+			this.m.Items.equip(::new(weapon));
+		}
+
+		if (this.m.Items.hasEmptySlot(this.Const.ItemSlot.Body))
+		{
+			local armor = ::Reforged.ItemTable.BanditArmorBasic.roll({
+				Apply = function ( _script, _weight )
+				{
+					local conditionMax = ::ItemTables.ItemInfoByScript[_script].ConditionMax;
+					if (conditionMax < 170 || conditionMax > 210) return 0.0;
+					return _weight;
+				}
+			})
+			this.m.Items.equip(::new(armor));
+		}
+
+		if (this.m.Items.hasEmptySlot(this.Const.ItemSlot.Head))
+		{
+			local helmet = ::Reforged.ItemTable.BanditHelmetBasic.roll({
+				Apply = function ( _script, _weight )
+				{
+					local conditionMax = ::ItemTables.ItemInfoByScript[_script].ConditionMax;
+					if (conditionMax < 140 || conditionMax > 180) return 0.0;
+					if (conditionMax >= 140 || conditionMax < 150) return _weight * 0.5;
+					return _weight;
+				}
+			})
+			this.m.Items.equip(::new(helmet));
+		}
+	}
+
+	function onSetupEntity()
+	{
+		local mainhandItem = this.getMainhandItem();
+		if (mainhandItem != null)
+		{
+			if (mainhandItem.isWeaponType(::Const.Items.WeaponType.Axe))
+	    	{
+	    		switch (mainhandItem.getID())
+	    		{
+	    			case "weapon.rf_battle_axe":
+	    				::Reforged.Skills.addPerkGroupOfEquippedWeapon(this, 4);
+	    				this.m.Skills.add(this.new("scripts/skills/perks/perk_rf_exploit_opening"));
+	    				this.m.Skills.add(this.new("scripts/skills/perks/perk_rf_rebuke"));
+	    				break;
+	    			case "weapon.woodcutters_axe":
+	    				::Reforged.Skills.addPerkGroupOfEquippedWeapon(this, 4);
+	    				this.m.Skills.add(this.new("scripts/skills/perks/perk_coup_de_grace"));
+	    				this.m.Skills.add(this.new("scripts/skills/perks/perk_crippling_strikes"));
+	    				this.m.Skills.add(this.new("scripts/skills/perks/perk_sundering_strikes"));
+	    				this.m.Skills.removeByID("actives.rf_bearded_blade");
```

Comment:
This removal of Bearded Blade is not reliable. As Bearded Blade is granted by the Axe Mastery perk, if he unequips and then re-equips an axe, he will get the Bearded Blade skill. 

I see you do this in some other entities too.

I currently can't think of a clean way to permanently make Bearded Blade unavailable to an entity, but I can take a look at that once I am back.

Reply:
I have created an issue (#154) for this and assigned it to myself. Once I am back home I can see how to do that. For now we can merge this in as is.